client/src/index.jsx

```diff
@@ -34,36 +33,53 @@ export default class App extends Component {
   constructor( props ) {
     super( props );
     this.state = {
+      adults: 1,
+      children: 0,
+      infants: 0,
+      totalGuests: 1,
+      listingData: null,
     };
   }
 
   componentDidMount() {
-    this.getPathname().then( () => {
-      this.getListingData();
+    this.getListingData();
+  }
+
+  onGuestButtonClick = ( guest, increment ) => {
+    const val = this.state[ guest ] + increment;
+    const { totalGuests } = this.state;
+    const total = totalGuests + increment;
+    const { maxGuests } = this.state.listingData;
+    // infants don't count towards the number of guests, but have a cap of 5
```

Comment:
I like your use of comments to make things clearer, but I think we would want to remove these before actually merging to master. For our project I think this is fine, but it's just something to keep in mind.

Reply:
You're right!